testsupport/tiers/checks.go

```diff
@@ -221,9 +221,9 @@ type base1nsTierChecks struct {
 
 func (a *base1nsTierChecks) GetNamespaceObjectChecks(_ string) []namespaceObjectsCheck {
 	checks := []namespaceObjectsCheck{
-		resourceQuotaComputeDeploy("20", "18Gi", "3", "18Gi"),
+		resourceQuotaComputeDeploy("30", "30Gi", "3", "30Gi"),
```

Comment:
it looks like you haven't increased the `limits.cpu` in the host operator PR

Reply:
Good catch! I initially didn't want to increase the cpu limit quota but later decided to do this after additional calculation. However I forgot to push it to the host PR :( 
Here we go: https://github.com/codeready-toolchain/host-operator/pull/1133/commits/998263a535739b4b053d04d888d1cf16b6a710bc